docs/how-to-provision-using-dps.md

```diff
@@ -216,19 +222,23 @@ namespace Samples.AdtIothub
 }
 ```
 
+Save the file and then re-publish your function app. For instructions on publishing the function app, see the [*Publish the app*](tutorial-end-to-end.md#publish-the-app) section of the end-to-end tutorial.
+
 ### Configure your function
 
-Next, you'll need to set environment variable in your function app from earlier, containing the reference to the Azure Digital Twins instance you've created. If you used the the end-to-end tutorial ([*Tutorial: Connect an end-to-end solution*](./tutorial-end-to-end.md)) your settings will already be configured, otherwise review both the setting up of the sample function app as well as assigning permissions to the function app.
+Next, you'll need to set environment variables in your function app from earlier, containing the reference to the Azure Digital Twins instance you've created.
+
+If you used the the end-to-end tutorial ([*Tutorial: Connect an end-to-end solution*](./tutorial-end-to-end.md)), your settings will already be configured.
+
+To set up the environment variable, you will need the Azure Digital Twins instance's **_host name_** from when you set up your instance. If you need to gather this value again, use [these instructions](how-to-set-up-instance-portal.md#verify-success-and-collect-important-values) from the setup article for finding them in the [Azure portal](https://portal.azure.com).
 
-You will need the following value from when you set up your instance. 
-If you need to gather this value again, use the link below to the corresponding section in the setup article for finding them in the [Azure portal](https://portal.azure.com).
-* Azure Digital Twins instance **_host name_** ([find in portal](../articles/digital-twins/how-to-set-up-instance-portal.md#verify-success-and-collect-important-values))
+Add the setting with this command:
 
 ```azurecli-interactive
 az functionapp config appsettings set --settings "ADT_SERVICE_URL=https://<Azure Digital Twins instance _host name_>" -g <resource group> -n <your App Service (function app) name>
 ```
 
-Additionally review the section 'Assign permissions to the function app' in the end-to-end tutorial ([*Tutorial: Connect an end-to-end solution*](./tutorial-end-to-end#assign-permissions-to-the-function-app)) for configuring Function app permissions and role assignment with Managed Identity.
+Additionally, review the section *Assign permissions to the function app* in the end-to-end tutorial ([*Tutorial: Connect an end-to-end solution*](./tutorial-end-to-end#assign-permissions-to-the-function-app)) for configuring function app permissions, and role assignment with Managed Identity.
```

Comment:
We can be more crisp about the ask-- Are we asking the reader to actually visit that article and perform the steps in it right now? Or just dropping a reference in case they want more information?

Reply:
Agree. The reader should actually visit the article and perform the steps (if not already done), because it is the only way to get access to the ADT instance and to get the sample running.